From 3849fcad3f72a7dd621c9a3dd3f446c40dc10a27 Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Sat, 09 May 2020 10:51:28 +0300
Subject: [PATCH] [SW-0] goya2: allow linking flash_image_offset

remove static from function arcboot_flash_image_offset so code running
from RAM of other places can link against  this function also.

Change-Id: Ic9eb68f8b2c34fb5aecedee1897719fae38eb0b6
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/app/arcboot_rom.c b/app/arcboot_rom.c
index b3fcf34..44a87a1 100644
--- a/app/arcboot_rom.c
+++ b/app/arcboot_rom.c
@@ -168,7 +168,7 @@
 
 }
 
-static uint32_t arcboot_flash_image_offset(uint8_t image_ptr)
+uint32_t arcboot_flash_image_offset(uint8_t image_ptr)
 {
 	uint32_t flash_image_offset;
 	uint8_t flash_image_switch;
